praw/models/reddit/mixins/__init__.py

```diff
@@ -118,12 +118,17 @@ def lock(self):
             API_PATH["lock"], data={"id": self.thing.fullname}
         )
 
-    def remove(self, spam=False):
+    def remove(self, mod_note="", reason_id=None, spam=False):
         """Remove a :class:`~.Comment` or :class:`~.Submission`.
 
+        :param reason_id: The removal reason ID.
```

Comment:
As-is this order doesn't match the parameter order. Once we settle on the order (likely, spam, mod_note, reason_id) then these _should_ also be in the same order.